data/ecosystems/a/avalanche.toml

```diff
@@ -116,6 +116,7 @@ sub_ecosystems = [
   "RocketX",
   "Roco Finance",
   "RUGenerous",
+  "Salvor",
```

Comment:
This ecosystem doesn't exist so you'll probably want to create a toml file for it if you want it to be a subecosystem of Avalanche, and then move the salvorio urls into that new toml file

Reply:
It's done, thanks!